.github/workflows/test_code_integrity.yml

```diff
@@ -46,7 +46,8 @@ jobs:
           --force
           --std=c++17
           --inline-suppr
-          --suppress=cppcheckError
+          --suppress=missingInclude
+          --suppress=missingIncludeSystem
```

Comment:
Fixes:

    [nofile:0]:(information),[unmatchedSuppression],Unmatched suppression: cppcheckError

And the 6000 warnings like this (is there a better way to allow checking for missing includes?):

    [src/ConvertInputFormat/main.cpp:51]:(information),[missingInclude],Include file: "EnergyPlus/InputProcessing/InputValidation.hh" not found.
    [src/ConvertInputFormat/main.cpp:52]:(information),[missingIncludeSystem],Include file: <embedded/EmbeddedEpJSONSchema.hh> not found. Please note: Cppcheck does not need standard library headers to get proper results.